pyanaconda/ui/gui/spokes/custom_storage.py

```diff
@@ -302,9 +302,7 @@ def refresh(self):
         self._default_file_system = self._device_tree.GetDefaultFileSystem()
 
         # Initialize the selected disks.
-        selected_disks = self._disk_selection.SelectedDisks
-        partitioned_devices = self._device_tree.GetPartitioned()
-        self._selected_disks = filter_disks_by_names(partitioned_devices, selected_disks)
```

Comment:
I am a little worried about the side effects of this change. Did it work before?

Reply:
In this situation, self._device_tree.GetPartitioned()  is used to check whether the disk format type is 'disklabel', which means that I can not use a formatted disk without partition table to install OS. I think it is unjustified.  In other cases, it truly works.